org.mozilla.thunderbird.app

```diff
@@ -0,0 +1 @@
+JSON=org.mozilla.thunderbird.json
```

Comment:
Sorry I meant this entire file isn't used by anything and can be removed.

Reply:
Oh, okay.